opencti-platform/opencti-front/src/private/components/analysis/malware_analyses/MalwareAnalysis.tsx

```diff
@@ -0,0 +1,156 @@
+import makeStyles from '@mui/styles/makeStyles';
+import Grid from '@mui/material/Grid';
+import { MalwareAnalysis_malwareAnalysis$key } from './__generated__/MalwareAnalysis_malwareAnalysis.graphql';
+import MalwareAnalysisDetails from './MalwareAnalysisDetails';
+import StixDomainObjectOverview from '../../common/stix_domain_objects/StixDomainObjectOverview';
+import SimpleStixObjectOrStixRelationshipStixCoreRelationships from '../../common/stix_core_relationships/SimpleStixObjectOrStixRelationshipStixCoreRelationships';
+import StixCoreObjectOrStixCoreRelationshipLastReports from '../reports/StixCoreObjectOrStixCoreRelationshipLastReports';
+import StixCoreObjectExternalReferences from '../external_references/StixCoreObjectExternalReferences';
+import StixCoreObjectLatestHistory from '../../common/stix_core_objects/StixCoreObjectLatestHistory';
+import StixCoreObjectOrStixCoreRelationshipNotes from '../notes/StixCoreObjectOrStixCoreRelationshipNotes';
+import Security from '../../../../utils/Security';
+import { KNOWLEDGE_KNUPDATE } from '../../../../utils/hooks/useGranted';
+import MalwareAnalysisEdition from './MalwareAnalysisEdition';
+import MalwareAnalysisPopover from './MalwareAnalysisPopover';
+import StixDomainObjectHeader from '../../common/stix_domain_objects/StixDomainObjectHeader';
+
+const useStyles = makeStyles(() => ({
+  gridContainer: {
+    marginBottom: 20,
+  },
+}));
+
+const MalwareAnalysisFragment = graphql`
+    fragment MalwareAnalysis_malwareAnalysis on MalwareAnalysis {
+        id
+        standard_id
+        entity_type
+        x_opencti_stix_ids
+        created
+        modified
+        created_at
+        revoked
+        confidence
+        product
+        submitted
+        result_name
+        createdBy {
+            ... on Identity {
+                id
+                name
+                entity_type
+            }
+        }
+        creators {
+            id
+            name
+        }
+        objectMarking {
+            edges {
+                node {
+                    id
+                    definition_type
+                    definition
+                    x_opencti_order
+                    x_opencti_color
+                }
+            }
+        }
+        objectLabel {
+            edges {
+                node {
+                    id
+                    value
+                    color
+                }
+            }
+        }
+        objectAssignee {
+            edges {
+                node {
+                    id
+                    name
+                    entity_type
+                }
+            }
+        }
+        status {
+            id
+            order
+            template {
+                name
+                color
+            }
+        }
+        workflowEnabled
+        ...MalwareAnalysisDetails_malwareAnalysis
+    }
+`;
+
+const MalwareAnalysis: FunctionComponent<{ data: MalwareAnalysis_malwareAnalysis$key }> = ({ data }) => {
```

Comment:
You can remove the Function component if you doesn't use Props interface and just write it like that : 

const MalwareAnalysis = ({ data }: { data: MalwareAnalysis_malwareAnalysis$key }) => {